Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -149,20 +149,20 @@ private extension CardPresentPaymentStore {
             // to the WCPay backend managed Payment Intent, which we need
             // to use to capture the payment.
             // This is always failing at this point. Commented out for now.
-            self.remote.captureOrderPayment(for: siteID, orderID: orderID, paymentIntentID: intent.id) { result in
-                switch result {
-                case .success(let intent):
-                    guard intent.status == .succeeded else {
-                        DDLogDebug("Unexpected payment intent status \(intent.status) after attempting capture")
-                        onCompletion(.failure(CardReaderServiceError.paymentCapture()))
-                        return
-                    }
-                    onCompletion(.success(receiptParameters))
-                case .failure(let error):
-                    onCompletion(.failure(error))
-                    return
-                }
-            }
+//            self.remote.captureOrderPayment(for: siteID, orderID: orderID, paymentIntentID: intent.id) { result in
```

Comment:
Commenting this out for now. It should be extracted out of this class, but we'll do that in #4030

Reply:
LOL 👍